apps/transport/lib/transport/gbfs_metadata.ex

```diff
@@ -115,25 +115,44 @@ defmodule Transport.GBFSMetadata do
     with {:feed_exists, true} <- {:feed_exists, not is_nil(feed_url)},
          {:ok, %HTTPoison.Response{status_code: 200, body: body}} <- cached_http_get(feed_url),
          {:ok, json} <- Jason.decode(body) do
-      not Enum.empty?(json["data"]["stations"])
+      json["data"]["stations"]
+      |> Enum.reject(& &1["is_virtual_station"])
+      |> Enum.count() > 0
```

Comment:
```suggestion
      |> Enum.empty?() == false
```

Reply:
J'ai hésité avec not `Enum.empty?` ou ce que j'ai mis et je trouve que ce que j'ai retenu est plus lisible.

Bref, tout est pareil.